phobos/cap-vars.yml

```diff
@@ -0,0 +1,54 @@
+---
+log_flavor: vm-osa-aio
+log_image: a4a9bc69-de27-4cec-903a-cf57be693738
+log_count: 1
+
+stor_flavor: ironic-rpc-storage-cap
+stor_image: 78856fa6-1de4-42d3-b545-466f6501e21f
+
+security_group: 58753c65-1804-4eee-8349-77f1c5573001
+external_network: ironic
+networks:
+  - net-name: "{{ external_network }}"
+
+fiobench_pgnum: 2048
+fiobench_size: 100G
+sata_bench: false
+ssd_bench: false
+bench_rgw_user_password: rgwbenchsecret
+bench_rgw_hummingbird_url:  https://github.com/troubling/hummingbird/releases/download/v1.0.0/hummingbird
```

Comment:
Likewise for this, I think this is a mistake though because we only specify the hummingbird url in the test-vars and it should probably be in group_vars/all/00-defaults.yml since we would want this to work in prod without specifying an additional var.